### DISCUSSION TITLE
Photos with just a year assigned

### DISCUSSION THREAD

**Discussion by Str1atum** [photoprism/photoprism#1970] [Q&A] [answered]
Photos which just have a year assigned (and no month or day) are not shown in the calendar view – is that expected behavior?
I have several old scanned images like from 1978 with a manually assigned year – would be great to find them in a "year" folder

**Comment by graciousgrey** [comment, marked as answer]
Yes, that's expected as there are only month folders in the calendar. To view all photos of a year, you can use the year filter on the normal search page :)
